index.html

```diff
@@ -75,22 +128,27 @@
           <div class="box">
             <div class="field is-grouped">
               <p class="control is-expanded">
-                <input class="input">
+                <label>
+                   get notification:
+                  <input class="input" name="notification">
+                </label>
               </p>
               <p class="control">
-                <a class="button is-info">Notify Me</a>
+                <a tabindex="0" tabindex="0" class="button is-info">Notify Me</a>
               </p>
             </div>
           </div>
         </div>
       </div>
     </div>
-  </div>
+  </header>
+
+  <main id="main">
 
-  <div class="section" id="about">
+  <section id="about" class="section" id="about">
     <div class="section-heading has-text-centered">
-      <div class="title is-2">About Us</div>
-      <div class="subtitle is-5">We of all trades, master of "some"</div>
+      <h2 class="title is-2">About Us</h2>
+      <h5 class="subtitle is-5">We of all trades, master of "some"</h5>
```

Comment:
You can't have `h5` right after `h2`. It should be `h3`. Run Lighthouse test to check.

Reply:
Done, thanks. I changed all h5 to h3 at headers of sections. Except for tiles.